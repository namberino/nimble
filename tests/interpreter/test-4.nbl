mut li = [];

li[1] = 1;
print(li);
