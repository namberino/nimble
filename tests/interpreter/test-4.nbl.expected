Index out of range
On line 3
